INSTALL

```diff
@@ -90,24 +82,26 @@ For example (M3ULCB case),
 ln -s rcar-proprietary-graphic-salvator-x-m3-xt-domd.tar.gz rcar-proprietary-graphic-m3ulcb-xt-domd.tar.gz
 ln -s rcar-proprietary-graphic-salvator-x-m3-xt-domu.tar.gz rcar-proprietary-graphic-m3ulcb-xt-domu.tar.gz
 
-7. Edit local.conf:
-
-workspace_base/build/conf/local.conf
+5. Run the build script for current stable release:
```

Comment:
Not sure you need to mention "stable release" here

Reply:
ok